tutorial/consuming_packages/the_flexibility_of_conanfile_py.rst

```diff
@@ -346,10 +346,48 @@ Windows or that you want to use the system's CMake installation instead of using
                 self.tool_requires("cmake/3.22.6")
 
 
+Use the generate() method to copy resources from packages
```

Comment:
The section is great, my only question is if it deserves to be part of the main thread of the tutorial. Is it an important use case, something that >50% of users need to learn the Conan basics? Or maybe it should be moved to elsewhere?

Reply:
That was exactly my question too... I opted for this because I thought that maybe it could be useful for someone checking what are the benefits of the conanfile.py vs the txt because this can't be done in the txt... but this could be moved to a complete example elsewhere...

Reply:
Yeah, I see... I think it can stay here, but maybe add a note clarifying that this isn't necessary to copy libraries, static or shared, as Conan generators and virtualenvs can use the libraries from their locations without copying it over and over.